README.md

```diff
@@ -711,3 +713,33 @@ cast send <SOURCE_MINTER_ADDRESS> --rpc-url avalancheFuji --private-key=$PRIVATE
 ```
 
 depending on whether you filled the [`SourceMinter.sol`](./src/cross-chain-nft-minter/SourceMinter.sol) contract with `Native (0)` or `LINK (1)` in step number 3.
+
+## CCT Standard
+
+The [Cross-Chain Token (CCT)](https://docs.chain.link/ccip/concepts/cross-chain-token/overview) standard offers a streamlined and decentralized approach to enabling token transfers across blockchains using Chainlink's Cross-Chain Interoperability Protocol (CCIP).
+
+Using CCT one can add support for cross-chain token transfers via CCIP for its own ERC20 token.
+
+This repository includes examples of ERC-20 tokens with:
+
+1. `getCCIPAdmin()` function in `MockERC20BurnAndMintToken` in [`CCIPv1_5ForkBurnMintPoolFork.t.sol`](./test/fork/CCIPv1_5BurnMintPoolFork.t.sol) file
+2. `owner()` function in `MockERC20TokenOwner` in [`CCIPv1_5LockReleasePoolFork.t.sol`](./test/fork/CCIPv1_5LockReleasePoolFork.t.sol) file
+
+To run CCT examples make sure you have the following environment variables set in your `.env` file:
+
+```shell
+ETHEREUM_SEPOLIA_RPC_URL=""
+BASE_SEPOLIA_RPC_URL=""
+```
+
+1. To run the CCT example with ERC20 token with `getCCIPAdmin()` function and Burn & Mint token pools, run the following command:
```

Comment:
"To run the CCT example with ERC20 token with"
---->
"To run the ERC20 CCT example with `getCCIPAdmin()`"